src/EnrollUserController.js

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+define([
+  'okta',
+  'models/EnrollUser',
+  'util/BaseLoginController',
+  'views/enrollUser/EnrollUserForm',
+  'views/shared/FooterWithBackLink',
+],
+function (
+  Okta,
+  EnrollUser,
+  BaseLoginController,
+  EnrollUserForm,
+  FooterWithBackLink,
+) {
+  var _ = Okta._;
+  return BaseLoginController.extend({
+    className: 'enroll-user',
+    initialize: function (options) {
```

Comment:
just realized about this... sorry this is late in the review. This should probably be in `fetchInitialData`, which is part of the `BaseLoginController`. That function is called when we call render on the controller, and we only render after `fetchInitialData` promise is fulfilled. This is similar to [RegistrationController](https://github.com/okta/okta-signin-widget/blob/master/src/RegistrationController.js)

So we can return either a promise that is resolved immediatly if we are not in unauthenticated, or we can return the `getEnrollFormData` promise.